data/fixtures/scopes/java/argument.formal.iteration.scope

```diff
@@ -0,0 +1,12 @@
+public class MyClass {
+    public void myFunk(int value, String name) { }
+}
+---
+
+[Range] = 1:23-1:45
+                         >----------------------<
+1|     public void myFunk(int value, String name) { }
+
+[Domain] = 1:4-1:50
+      >----------------------------------------------<
+1|     public void myFunk(int value, String name) { }
```

Comment:
this is nice. do we do this in typescript? we should prob document that this is the desired behaviour in the scope info if we don't

Reply:
Yes we do
https://github.com/cursorless-dev/cursorless/blob/fd229a29fd36fbfb33c4e2d87afd88fad24ae64b/data/fixtures/scopes/javascript.core/argument.formal.iteration.scope

Description updated